app/assets/stylesheets/stream_element.scss

```diff
@@ -72,15 +107,25 @@
 }
 
 .stream-element {
-  background-color: $framed-background;
   padding: 10px;
 
+  .post-wrapper {
+    border: 1px solid #ddd;
```

Comment:
- Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Properties should be ordered background-color, border, border-radius